apps/passport-server/src/services/generic-issuance/pipelines/PretixPipeline.ts

```diff
@@ -244,7 +245,9 @@ export class PretixPipeline implements BasePipeline {
   public async start(): Promise<void> {
     // On startup, the pipeline definition may have changed, and manual tickets
     // may have been deleted. If so, clean up any check-ins for those tickets.
-    await this.cleanUpManualCheckins();
+
+    // @todo: turn this back on after eth berlin
+    // await this.cleanUpManualCheckins();
```

Comment:
important we don't forget this

Reply:
linear ticket?

Reply:
https://linear.app/0xparc-pcd/issue/0XP-859/turn-on-manual-ticket-cleanup-post-eth-berlin